raiden-dapp/src/components/NoTokens.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-container class="new-token">
+    <v-col class="new-token__button">
+      <v-btn fab color="primary" @click="navigateToTokenSelect()">
+        <v-icon large>mdi-plus</v-icon>
+      </v-btn>
+    </v-col>
+    <v-col class="new-token__header">
+      {{ $t('tokens.connect-new') }}
+    </v-col>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import NavigationMixin from '@/mixins/navigation-mixin';
+
+@Component({})
+export default class NoTokens extends Mixins(NavigationMixin) {}
+</script>
+
+<style lang="scss" scoped>
+.new-token {
+  text-align: center;
```

Comment:
Can you move the button and header parts here?
as `&__button ` and `&__header`?

Reply:
Done